source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,89 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the replication option you use for the site.
+
+.. list-table:: 
+   :widths: 25 75
+   :width: 100%
+
+   * - Site Replication
+     - Total restoration from the healthy peer site(s)
+   * - Bucket Replication
+     - Data restoration of objects and metadata from a healthy remote location for each bucket configured for replication
+   * - :mc-cmd:`mc mirror`
+     - Data restoration of objects only (no metadata) from a healthy remote location
```

Comment:
it is inaccurate to say `no metadata` is restored in `mc mirror` - it will restore other metadata except VersionID, ModTime. Difference b/w bucket replication and `mc mirror` is that replication preserves version history and all metadata whereas mirror is more useful for unversioned buckets OR where they care only to restore the latest copy

Reply:
I appreciate the clarification and correction. 
Removed the `(no metadata)` from the sentence. I think the Mirroring section at the bottom of the page adequately addresses the use cases.